projects/core/src/styles/theme.dark.scss

```
@@ -37,6 +37,7 @@
   --cds-alias-typography-link-color-visited-hover: #{$cds-global-color-lavender-200};
 
   --cds-alias-object-app-background: #{$cds-global-color-construction-1000};
+  --cds-alias-object-container-backdrop-background: hsla(0, 0%, 0%, 0.6);
```

You may use `$cds-alias-object-opacity-300` instead of hard coding it. 

```suggestion
  --cds-alias-object-container-backdrop-background: #{$cds-alias-object-opacity-300};
```

---

Added in 19e6b52